tasks/terraform-cli/src/tests/steps/task-context.steps.ts

```diff
@@ -160,6 +161,27 @@ export class TaskContextSteps {
         this.ctx.skipExistingWorkspace = true;
     }
 
+    @given("subcommand is {string}")
+    public subCommand(subCommand: string){

```

Comment:
can we rename to make state specific?
```suggestion
    public stateSubCommandIs(subCommand: string){
```